lldb/source/Plugins/Language/CPlusPlus/LibCxx.cpp

```diff
@@ -456,9 +464,13 @@ ExtractLibcxxStringInfo(ValueObject &valobj) {
   if (!l)
     return {};
 
-  StringLayout layout = l->GetIndexOfChildWithName("__data_") == 0
-                            ? StringLayout::DSC
-                            : StringLayout::CSD;
+  auto index_or_err = l->GetIndexOfChildWithName("__data_");
+  if (!index_or_err)
+    LLDB_LOG_ERROR(GetLog(LLDBLog::Types), index_or_err.takeError(), "{0}");
+  return {};
```

Comment:
This is supposed to be inside the if statement body right?

Reply:
Yes, it was actually the root cause of some test failures as well...